css/main.css

```diff
@@ -20,11 +20,35 @@ img:hover {
 /* END sprint 1  */
 
 
+/* icon legend */
+
+.legend{
+   text-align: center;
+   align-items: center;
+   align-content: center;
+   font-size: 2em;
+  
+ 
+}
+
+table, th, td {
+   border: 2px solid black;
+   align-content: center;
 
+   
+ 
+ }
 
+ table{
+    width: 100%;
+ }
 
 
 
 
+.legend img{
+max-width: 120px;
+max-height: 120px;
```

Comment:
Some of the icons are displaying with different shapes (square vs rectangle). Should they be the same size and shape for consistency? If so, should this be done by cropping the image files or controlling it in the css?

Reply:
yeah, I plan to crop the images to have same sizes so they dont mess up later on.

Reply:
> 
> 
> Some of the icons are displaying with different shapes (square vs rectangle). Should they be the same size and shape for consistency? If so, should this be done by cropping the image files or controlling it in the css?

yeah, I plan to crop the images to have same sizes so they dont mess up later on.